src/Compilers/Lua/Portable/Syntax/Syntax.xml

```diff
@@ -1,1742 +1,1997 @@
+        </TypeComment>
+        <FactoryComment>
+            <summary>
+                Creates a new <see cref="TypeUnionSyntax" /> node.
+            </summary>
+        </FactoryComment>
+        <Kind Name="TypeIntersectionSyntax" />
+        <Field Name="Left" Type="TypeBindingSyntax">
+            <PropertyComment>
+                <summary>The type on the left side of the operator.</summary>
+            </PropertyComment>
+        </Field>
+        <Field Name ="AmpersandToken" Type="SyntaxToken">
+            <PropertyComment>
+                <summary>The pipe at the middle of the operator.</summary>
+            </PropertyComment>
+            <Kind Name="AmpersandToken" />
+        </Field>
+        <Field Name="Right" Type="TypeBindingSyntax">
+            <PropertyComment>
+                <summary>The type on the right side of the operator.</summary>
+            </PropertyComment>
+        </Field>
+    </Node>
+
+    <Node Name="TypeParameterListSyntax" Base="LuaSyntaxNode">
+        <TypeComment>
+            <summary>Represents a type parameter list</summary>
+        </TypeComment>
+        <FactoryComment>
+            <summary>
+                Creates a new <see cref="TypeParameterListSyntax" /> node.
+            </summary>
+        </FactoryComment>
+        <Kind Name="TypeParameterListSyntax" />
+        <Field Name="OpeningToken" Type="SyntaxToken">
+            <PropertyComment>
+                <summary>The parenthesis at the beginning of the operator.</summary>
+            </PropertyComment>
+            <Kind Name="LessThanToken" />
+        </Field>
+        <Field Name="Names" Type="SeparatedSyntaxList&lt;SyntaxToken&gt;">
+            <PropertyComment>
+                <summary>A list of types.</summary>
+            </PropertyComment>
+        </Field>
+        <Field Name="ClosingToken" Type="SyntaxToken">
+            <PropertyComment>
+                <summary>The parenthesis at the beginning of the operator.</summary>
+            </PropertyComment>
+            <Kind Name="GreaterThanToken" />
+        </Field>
+    </Node>
+
+    <Node Name="TypeArgumentListSyntax" Base="LuaSyntaxNode">
+        <TypeComment>
+            <summary>Represents a type argument list</summary>
+        </TypeComment>
+        <FactoryComment>
+            <summary>
+                Creates a new <see cref="TypeArgumentListSyntax" /> node.
+            </summary>
+        </FactoryComment>
+        <Kind Name="TypeArgumentListSyntax" />
+        <Field Name="OpeningToken" Type="SyntaxToken">
+            <Kind Name="LessThanToken" />
+        </Field>
+        <Field Name="Arguments" Type="SeparatedSyntaxList&lt;TypeSyntax&gt;">
+            <PropertyComment>
+                <summary>A list of types.</summary>
+            </PropertyComment>
+        </Field>
+        <Field Name="ClosingToken" Type="SyntaxToken">
+            <Kind Name="GreaterThanToken" />
+        </Field>
```

Comment:
Here the field name can be the token kind name.
It's also missing the `PropertyComment`.
```suggestion
        <Field Name="GreaterThanToken" Type="SyntaxToken">
            <PropertyComment>
                <summary>Gets the <c>&gt;</c> token.</summary>
            </PropertyComment>
            <Kind Name="GreaterThanToken" />
        </Field>
```